### PR TITLE
irmin-bench: improve CLI

### PR DESCRIPTION
**Pull request by Ngoguey42**
extracted from #1332 

Series of small normalisations on the CLI arguments, before bringing back the summary 🥳

### CODE REVIEW

File: bench/irmin-pack/trace_collection.ml [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+(** Trace file construction.
+
+    This file is also meant to be used from Tezos. OCaml version 4.09 and the
+    32bit architecture should be supported.
+
+    A module [Make_replayable] has yet to be implemented. *)
+
+let ( >>= ) = Lwt.Infix.( >>= )
+let ( >|= ) = Lwt.Infix.( >|= )
```
Comment (samoht) [inline bench/irmin-pack/trace_collection.ml:25]:
```suggestion
open Lwt.Infix
```

File: bench/irmin-pack/trace_collection.ml [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+let ( let* ) = ( >>= )
+let ( let+ ) = ( >|= )
```
Comment (samoht) [inline bench/irmin-pack/trace_collection.ml:27]:
```suggestion
open Lwt.Syntax
```

File: bench/irmin-pack/trace_collection.ml (hunk 1 above)
Comment (Ngoguey42) [reply bench/irmin-pack/trace_collection.ml:25]: 👍 I'll change this in a following PR

**Comment by samoht** [comment]
Could you review @icristescu

File: bench/irmin-pack/dune [hunk 3]
```diff
@@ -27,13 +27,14 @@
  (modules trace_common trace_definitions trace_collection)
  (preprocess
   (pps ppx_repr ppx_deriving.enum))
- (libraries irmin irmin-pack unix lwt repr ppx_repr mtime mtime.clock.os))
+ (libraries irmin irmin-pack unix lwt repr ppx_repr bentov mtime printbox
```
Comment (icristescu) [inline bench/irmin-pack/dune:30]: you are not using `printbox` in this PR, no?

File: bench/irmin-pack/tree.ml [hunk 4]
```diff
@@ -775,6 +794,16 @@ let keep_stat_trace =
   in
   Arg.(value @@ flag doc)
 
+let empty_blobs =
+  let doc =
+    Arg.info
+      ~doc:
+        "Whether or not the blobs added to the store should be the empty \
+         sting, during trace replay. This greatly increases the replay speed."
```
Comment (icristescu) [inline bench/irmin-pack/tree.ml:802]:
```suggestion
         string, during trace replay. This greatly increases the replay speed."
```

**Review by icristescu — APPROVED** [review]
Thanks, looks good.

File: bench/irmin-pack/dune (hunk 3 above)
Comment (Ngoguey42) [reply bench/irmin-pack/dune:30]: 👍 I'll move this out of the PR

**Pull request merged**